src/test/java/networkbook/logic/commands/AddCommandTest.java

```diff
@@ -61,6 +63,26 @@ public void execute_noFieldSpecifiedUnfilteredList_success() {
         CommandTestUtil.assertCommandSuccess(addCommand, model, expectedMessage, expectedModel);
     }
 
+    @Test
+    public void execute_addPriorityToPersonWithPriority_commandException() {
+        CommandTestUtil.showPersonAtIndex(model, TypicalIndexes.INDEX_FIRST_PERSON);
+
+        AddCommand addCommand = new AddCommand(TypicalIndexes.INDEX_FIRST_PERSON,
+                new EditPersonDescriptorBuilder().withPriority(CommandTestUtil.VALID_PRIORITY_AMY).build());
+
+        String expectedMessage = AddCommand.MESSAGE_MULTIPLE_UNIQUE_FIELD;
+
+        try {
+            addCommand.execute(model); // person would have priority
+            addCommand.execute(model); // add priority to the person again
+            fail();
+        } catch (CommandException e) {
+            assertEquals(expectedMessage, e.getMessage());
```

Comment:
Please refactor to make use of `networkbook.logic.commands.CommandTestUtil.assertCommandFailure` instead.